gym/envs/classic_control/acrobot.py

```diff
@@ -171,6 +171,7 @@ def reset(self, *, seed: Optional[int] = None, options: Optional[dict] = None):
 
     def step(self, a):
         s = self.state
+        assert s is not None, "Call reset before using AcrobotEnv object."
```

Comment:
Isn't this enforced through a wrapper already? Or is this to make type checkers happy?

Reply:
yes, this is to make typechecker happy. It can't deduce the wrapper verification.